feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -60,12 +69,12 @@ func TestBasic(t *testing.T) {
 	}
 	isisRoot := session.ISISPath()
 	port1ISIS := isisRoot.Interface(ts.DUTPort1.Name())
-	if err := check.Equal(isisRoot.Global().Instance().State(), session.ISISName).AwaitFor(time.Second, ts.DUTClient); err != nil {
+	if err := check.Equal(isisRoot.Global().Instance().State(), session.ISISName).AwaitFor(time.Second*5, ts.DUTClient); err != nil {
 		t.Fatalf("IS-IS failed to configure: %v", err)
 	}
 	// There might be lag between when the instance name is set and when the
 	// other parameters are set; we expect the total lag to be under 5s
```

Comment:
Comment doesnt seem to match with what you are proposing here. Can you please update?

Reply:
updated , Thank you.